src/test/java/com/thealgorithms/maths/ComplexNumberUtilTest.java

```diff
@@ -0,0 +1,220 @@
+package com.thealgorithms.maths;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
```suggestion
import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertNotEquals;
import static org.junit.jupiter.api.Assertions.assertThrows;
import static org.junit.jupiter.api.Assertions.assertTrue;
import static org.junit.jupiter.api.Assertions.fail;
```

The plan (#5119) is to `AvoidStarImport`.